### ISSUE TITLE
Invalid expression error when opening any Python file in Vim 7.4

### ISSUE THREAD

**Issue by abbottc** [dbsr/vimpy#3]
I just installed vimpy to give it a try, but got an error.  I've looked at the code and tried to figure out the cause, but I'm stumped.  This is the traceback:

```
Error detected while processing /home/self/.vim/bundle/vimpy/ftplugin/python/vimpy.vim:
line   28:
Traceback (most recent call last):
File "<string>", line 4, in <module>
vim.error: invalid expression
E121: Undefined variable: let
E15: Invalid expression: let s:have_flake = 0
```

What's additionally odd is that I have pyflakes installed, so I would expect that line 28 would not get executed.

**Comment by dbsr** [comment]
hi abbotc, as you might have seen I haven't worked on vimpy for quite a while.

have you compiled vim with python enabled?

**Comment by abbottc** [comment]
Yes, python support is enabled.  I've also verified this error still occurs with a minimum vimrc file, so it's not due to a conflict with another plugin.

One thing I noticed is that the vim module isn't imported before it's used on [line 27](https://github.com/dbsr/vimpy/blob/master/ftplugin/python/vimpy.vim#L27).  But, that doesn't seem to relate to the error I'm seeing.  Although I'm proficient in Python, I'm not very familiar with VimL.  So, if it's the case that the vim module doesn't require a Python import statement for some reason, then maybe that doesn't matter.  Nonetheless, I did try adding "import vim" before Line 27, but there was no change to the error.

Since you haven't worked on the project in a while, are you able to verify that it works with Python 2.7.6 and Vim 7.4, or is that an unknown at this point?

**Comment by dbsr** [comment]
❯ python -V
Python 2.7.9

vim --version
VIM - Vi IMproved 7.4 (2013 Aug 10, compiled Mar 28 2015 09:07:17)

It's a weird error and one I have never seen before. Maybe it's a config related bug? Eg, have you tried using the plugin on a clean install?

I did some googling and found a couple (maybe) related issues for other plugins:

https://github.com/gmarik/Vundle.vim/issues/399

https://github.com/gmarik/Vundle.vim/issues/515

My (uneducated) guess is it has something to do with paths and / or windows. What's  your OS?

**Comment by abbottc** [comment]
The issue can be closed.  I ran some updates yesterday on Vim plugins, and today after reading your post and doing some re-testing, I see that the issue is no longer present.  My guess is that it was a compatibility problem with Vundle, because I had previously tested and seen the error with a clean Vim install with only the Vundle and vimpy plugins.  Plus, Vundle was one of the plugins I updated yesterday, implying maybe Vundle fixed a bug which affected vimpy.  Thanks for your responsiveness and research into the issue!
